content/docs/rfcs/27/README.md

```diff
@@ -0,0 +1,98 @@
+---
+slug: 27
+title: 27/WAKU2-PEERS
+name: Waku v2 Client Peer Management Recommendations
+status: raw
+editor: Hanno Cornelius <hanno@status.im>
+contributors:
+---
+
```

Comment:
Curious: what's the overlap between this and libp2p specs, if any? I.e. not just nim-libp2p but libp2p at a spec level. Are there some things that we can defer/point to upstream? Or is it basically underspecified?

Reply:
Yeah, very little overlap because it's underspecified in `libp2p`. They do recommend that peers keep [a peer store](https://docs.libp2p.io/reference/glossary/#peer-store), but not much more.

Each of the different implementations (e.g. [go](https://github.com/libp2p/go-libp2p-peerstore) and [rust](https://github.com/libp2p/js-libp2p/tree/master/src/peer-store)) does implement a peer store more or less according to what is recommended here (minus Waku v2 specific things like `Connectedness` and `DisconnectTime`).

Peer management is wholly underspecified, so I've focused on what was necessary for Waku v2.